types/es-abstract/GetIntrinsic.d.ts

```diff
@@ -117,8 +117,8 @@ declare namespace GetIntrinsic {
         '%SyntaxError%': SyntaxErrorConstructor;
         '%SyntaxErrorPrototype%': SyntaxError;
         '%ThrowTypeError%': () => never;
-        '%TypedArray%': unknown;
-        '%TypedArrayPrototype%': unknown;
+        '%TypedArray%': any;
+        '%TypedArrayPrototype%': any;
```

Comment:
why is anything `any`, when DT now requires a TS version that supports `unknown`?

Reply:
Because of ergonomics of consuming these types from **JavaScript** code.

This is also why the `GetIntrinsic` fallback overload return type was changed to `any`.

Reply:
Can you elaborate? JS code doesn't need to care about types; and either way, if `any` passes but `unknown` doesn't, you have a bug.

Reply:
Currently, `GetIntrinsic("%Reflect.apply%", true)` returns `unknown`, so it needs to be type cast using:
```js
const reflectApply = /** @type {typeof Reflect.apply | undefined} */ (GetIntrinsic("%Reflect.apply%", true));
```

---

By using `any`, one can do:
```js
/** @type {typeof Reflect.apply | undefined} */
const reflectApply = GetIntrinsic("%Reflect.apply%", true);
```

or even:
```js
const reflectApply = GetIntrinsic("%Reflect.apply%", true);
```

Which is what you get without typings.

Reply:
i can see why it'd be nice to have the typings overload on the literal string input, and return the proper type - but i'm still not understanding any `any` is better than `unknown`, and if you're using jsdoc in your JS, the whole point is to force the types to be correct.

Reply:
I’ve implemented that in <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45117>